src/main/java/com/phonebookservice/model/PhoneNumber.java

```diff
@@ -0,0 +1,19 @@
+package main.java.com.phonebookservice.model;
+
+public class PhoneNumber {
+    private String label;
+    private String number;
+
+    /**
+     * phone number constructor.
+     *
+     * @param label  the label.
+     * @param number the phone number.
+     */
+    public PhoneNumber(final String label, final String number) {
+        super();
```

Comment:
why do you call `super` here ? 

PhoneNumber doesn't extends from any class, what constructor `super` call here ?